linux_os/guide/system/permissions/restrictions/sysctl_kernel_unprivileged_bpf_disabled_accept_default/rule.yml

```diff
@@ -0,0 +1,80 @@
+documentation_complete: true
+
+prodtype: rhel9
+
+title: 'Disable Access to Network bpf() Syscall From Unprivileged Processes'
+
+description: |-
+    To prevent unprivileged processes from using the <code>bpf()</code> syscall
+    the <code>kernel.unprivileged_bpf_disabled</code> kernel parameter must
+    be set to <code>1</code> or <code>2</code>.
+
+    Writing <code>1</code> to this entry will disable unprivileged calls to <code>bpf()</code>; once
+    disabled, calling <code>bpf()</code> without <code>CAP_SYS_ADMIN</code> or <code>CAP_BPF</code> will return <code>-EPERM</code>.
+    Once set to <code>1</code>, this can't be cleared from the running kernel anymore.
+
+    Writing <code>2</code> to this entry will also disable unprivileged calls to <code>bpf()</code>,
+    however, an admin can still change this setting later on, if needed, by
+    writing <code>0</code> or <code>1</code> to this entry.
+
+    {{{ describe_sysctl_option_value(sysctl="kernel.unprivileged_bpf_disabled", value="1") }}}
```

Comment:
In the above description, you taslk about values of "1" or "2". But here you describe only value "1".